sparql/kidney.rq

```diff
@@ -226,15 +115,17 @@ WHERE {
   ?s rdfs:label ?s_label .
   { VALUES ?p
     { 
-      rdfs:subClassOf
       part_of: 
       overlaps: 
```

Comment:
Remove overlaps.  See ticket for why.

Reply:
Also - needs the redundant graph + closure graph - again see ticket.   (If combining graphs is hard - I think it will also work fine with no graph restrictions at all)

Reply:
Sorry, I haven't seen (notified) the edit in the spec.

Reply:
I changed it this morning in the first comment on the ticket.  Sorry - should have pinged you to notify you of the change.